### PR TITLE
chore(jetbrans-gateway): update JetBrains IDEs

### PR DESCRIPTION
**Pull request by matifali**
1. Updated IDEs to the latest available version
2. Remove the Community editions as they are not supported by the gateway.

### CODE REVIEW

File: jetbrains-gateway/main.tf [hunk 1]
```diff
@@ -58,37 +58,27 @@ locals {
...
     },
     "PY" = {
       icon  = "/icon/pycharm.svg",
       name  = "PyCharm Professional",
-      value = jsonencode(["PY", "232.9559.58", "https://download.jetbrains.com/python/pycharm-professional-2023.2.1.tar.gz"])
-    },
-    "PC" = {
-      icon  = "/icon/pycharm.svg",
-      name  = "PyCharm Community",
-      value = jsonencode(["PC", "232.9559.58", "https://download.jetbrains.com/python/pycharm-community-2023.2.1.tar.gz"])
```
Comment (stirby) [inline jetbrains-gateway/main.tf:86]: If we're removing support for PyCharm Community entirely we should update the README to reflect that.

**Review by stirby — CHANGES_REQUESTED** [review]
One comment on updating the README, looks good!

File: jetbrains-gateway/main.tf (hunk 1 above)
Comment (matifali) [reply jetbrains-gateway/main.tf:86]: It was added as a mistake and does not actually work with the gateway. I will update README.md

**Review by stirby — APPROVED** [review]
👍

**Pull request merged**